src/nplinker/pairedomics/downloader.py

```diff
@@ -85,15 +82,14 @@ def __init__(self, platform_id, force_download=False, local_cache = None):
 
             if gnps_massive_id == platform_id:
                 self.pairedomics_id = pairedomics_id
-                logger.debug(
-                    'platform_id {} matched to pairedomics_id {}'.format(
-                        self.gnps_massive_id, self.pairedomics_id))
+                logger.debug('platform_id %s matched to pairedomics_id %s',
```

Comment:
In PR #131 , I disabled the warning about `Use %s formatting in logging functions`. So it's fine if you have changed f-string to `%s`, otherwise using f-string is better for readability.

Reply:
Okay if you prefer I can change it to f-string, no problem. Maybe is better to have them uniform?